sql/src/test/java/io/crate/integrationtests/MetricsITest.java

```diff
@@ -22,11 +22,18 @@
 
 package io.crate.integrationtests;
 
+import io.crate.execution.engine.collect.stats.JobsLogService;
+import io.crate.execution.engine.collect.stats.JobsLogs;
+import io.crate.expression.reference.sys.job.JobContextLog;
+import org.elasticsearch.test.ESIntegTestCase;
 import org.hamcrest.Matchers;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
+import static org.hamcrest.core.Is.is;
+
+@ESIntegTestCase.ClusterScope(numClientNodes = 0, supportsDedicatedMasters = false)
```

Comment:
@seut we can also remove this configuration and use `getInstances()` instead of `getDataNodeInstances()` here: https://github.com/crate/crate/pull/7496/files#diff-9086f784bddb1aa09ea73633559813afR97

Reply:
@matriv yes would prefer that